src/components/Delegation/dashboard/DelegatedStakepoolInfo.js

```diff
@@ -40,8 +39,24 @@ const messages = defineMessages({
     id: 'components.delegationsummary.delegatedStakepoolInfo.copied',
     defaultMessage: '!!!Copied!',
   },
+  unknownPool: {
+    id: 'components.delegationsummary.delegatedStakepoolInfo.unknownPool',
+    defaultMessage: '!!!Unknown pool!',
```

Comment:
There's an `!` here whereas there isn't one in `en-US.json`. Probably we don't want the exclamation mark